packages/libs/components/src/plots/BipartiteNetwork.tsx

```diff
@@ -47,6 +60,8 @@ export interface BipartiteNetworkProps {
   labelTruncationLength?: number;
   /** Additional error messaging to show when the network is empty */
   emptyNetworkContent?: ReactNode;
+  /** Entries for the actions that appear in the menu when you click a node */
+  getNodeActions?: (nodeId: string) => NodeAction[];
```

Comment:
This is totally a nit but could we change the name slightly to make it more clear that these are the action options in the menu? Before I got here, i expected this to have how nodes get clicked and hovered and all that jazz as "actions". Maybe "getNodeMenuActions"? Just a suggestion!

Reply:
Sure, that sounds good to me!